src/Parser/FrostMage/Modules/Features/IcicleTracker.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class IcicleTracker extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  IciclesWasted = 0;
+
+  on_initialized() {
+	const hasGlacialSpike = this.combatants.selected.hasTalent(SPELLS.GLACIAL_SPIKE_TALENT.id);
+	this.active = hasGlacialSpike;
```

Comment:
This can be shortened to `this.active = this.combatants.selected.hasTalent(SPELLS.GLACIAL_SPIKE_TALENT.id);`

Reply:
Ill get that fixed.